Resources/Prototypes/Entities/Mobs/Species/reptilian.yml

```
@@ -93,17 +103,35 @@
   components:
   - type: HumanoidAppearance
     species: Reptilian
-    hideLayersOnEquip:
+    hideLayersOnEquip: # WWDP
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)